### PR TITLE
Add releasing instructions

### PR DESCRIPTION
**Pull request by jose5918**
Adding some releasing instructions. Not sure who has permissions to test this, but it should be similar release process to tf-operator 

@ankushagarwal 
@jlewi

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pytorch-operator/28)

### CODE REVIEW

File: releasing.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+Permissions
+
+	* You need to be a member of release-team@kubeflow.org to have access to the GCP
+	  resources used for releasing.
+
+	* You need write permissions on the repository to create a release branch.
+
+Use the GitHub UI to cut a release branch
+	* Name the release branch v{MAJOR}.${MINOR}-branch
```
Comment (ankushagarwal) [inline releasing.md]: Formatting issue

File: releasing.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Releasing the TFJob operator
```
Comment (ankushagarwal) [inline releasing.md]: Replace TFJob throughout the README

**Comment by jose5918** [comment]
@gaocegege Would you take a look?

**Comment by jose5918** [comment]
I signed it!

**Pull request merged**